atoms/input/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/index.css" />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="/utils/prism.css" />
+    <title>thisUI</title>
+  </head>
+  <body data-prismjs-copy-timeout="500">
+    <header class="tuid__head">
+      <nav class="tuid__nav--top"></nav>
+    </header>
+
+    <div
+      class="tuid__drawer tui__dis--none invis tui__scrollbar--invisible"
+    ></div>
```

Comment:
Just a suggestion, use prettier for good indentation

Reply:
@romabulani Thanks for suggesting I have the extension installed and it automatically breaks this into different lines which I probably need to see, if I can configure it to stop doing this unintentional line break.